MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/ComposeContentViewController.swift

```diff
@@ -207,8 +207,9 @@ extension ComposeContentViewController {
             self.tableView.contentInset.bottom = padding - self.view.safeAreaInsets.bottom
             self.tableView.verticalScrollIndicatorInsets.bottom = padding - self.view.safeAreaInsets.bottom
             UIView.animate(withDuration: 0.3) {
+                guard let window = self.view.window else { return }
                 // ref: https://developer.apple.com/documentation/uikit/uiresponder/1621578-keyboardframeenduserinfokey
-                let localKeyboardFrame = self.view.convert(endFrame, from: self.view.window!.screen.coordinateSpace)
+                let localKeyboardFrame = self.view.convert(endFrame, from: window.screen.coordinateSpace)
```

Comment:
Great, that you found that! What I didn't understand Why are we using `window.screen.coordinateSpace` here and not `view.coordinateSpace`? But then I ran the app on the iPad Simulator and found the answer.

Reply:
Yep!